Makefile

```diff
@@ -161,6 +161,11 @@ prepackaged-plugins: ## Populate the prepackaged-plugins directory
 		curl -f -O -L https://plugins-store.test.mattermost.com/release/$$plugin_package.tar.gz.sig; \
 	done
 
+prepackaged-binaries: ## Populate the prepackaged-binaries to the bin directory
+	@echo Downloading prepackaged binaries
+	curl -f -O -L https://github.com/mattermost/mmctl/releases/latest -o $(DIST_PATH)/bin/mmctl
```

Comment:
If we plan to use this target to create the packages, always fetching the latest version would cause problems when releasing server's dot versions (if we have to release e.g. 5.20 and 5.19.1, both would fetch and bundle `mmctl` 5.20). My suggestion would be to either directly get the version from the source code or to accept a version parameter with `latest` as the default to supply in case we build an "old" version

Reply:
That is a good point - the packaging is completed via the build/release.mk file targets. During a release there should be a release-5.20 branch in both repos - I might be able to use that info to request the correct mmctl version.